### DISCUSSION TITLE
Map display does not keep up with position

### DISCUSSION THREAD

**Discussion by mp13mp13** [osmandapp/OsmAnd#19198] [Maps Related]
https://drive.proton.me/urls/F949Y814ZW#GyK1PxawS7MK
In the link, a video showing the problem.

Device : e3q
Brand : samsung
Manufacturer : samsung
Model : SM-S928B
Product : e3qxeea
Build : UP1A.231005.007.S928BXXU1AXB5
Version : 14
App Version : OsmAnd+
Apk Version : 4.6.13 4613

Screen:
https://drive.proton.me/urls/HJVFYF7W9M#Le4OUXTSWPYG

**Comment by DmitryAlexei** [comment]
Hello, @mp13mp13
Does your issue persists on 4.7.10 release?

**Reply by mp13mp13** [reply]
I'll check it out at the earliest opportunity and let you know.

**Reply by mp13mp13** [reply]
I didn't notice the map jamming significantly. It's moving smoothly.
I made the inspection on foot, so the speed of movement was low.

**Reply by DmitryAlexei** [reply]
Thanks for testing!
